demos/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Demo: MyComponent</title>
+
+	<link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.css">
+
+	<script src="../node_modules/closure-templates/soyutils.js"></script>
+	<script src="../build/globals/myComponent.js"></script>
```

Comment:
Once you make the change I mentioned on the `gulpfile.js` file, you'll need to update this to include `../build/globals/layout.js` instead of `../build/globals/myComponent.js`.